### PR TITLE
Remarriage and number of FAM records

### PR DESCRIPTION
**Pull request by tychonievich**
I'm not sure if I put this in the right place. I put it under the `FAM` tag description, but perhaps we should move it to the `FAMILY_RECORD` description instead?

### CODE REVIEW

File: specification/gedcom.md [hunk 1]
```diff
@@ -2663,6 +2663,23 @@ Depending on the maintaining authority, an `EXID` may be a unique identifier for
...
+can be represented either as a single `FAM` with multiple events (`MARR`, `DIV`, etc)
+or as a separate `FAM` for each time together.
+Some user interfaces may display these two in different ways
+and the two admit different semantics in sourcing.
+Neither structure necessarily indicates this kind of temporary separation:
+a single `FAM` with two `MARR` with distinct dates might represent uncertainty about dates
+and a pair of `FAM` with same spouses might be the result of merging multiple files.
+
+Implementers should support both representations,
+and should to chose between them based on user input or other context beyond that provided in the datasets themselves.
```
Comment (tychonievich) [inline specification/gedcom.md:2680]:
```suggestion
and should choose between them based on user input or other context beyond that provided in the datasets themselves.
```

**Review by tychonievich — COMMENTED** [review]
typo

File: specification/gedcom.md [hunk 2]
```diff
@@ -2663,6 +2663,23 @@ Depending on the maintaining authority, an `EXID` may be a unique identifier for
 
 See `FAMILY_RECORD`
 
+:::note
+The common case is that each couple has one `FAM` record,
+but that is not always the case.
+
+A couple that separates and then gets together again
+can be represented either as a single `FAM` with multiple events (`MARR`, `DIV`, etc)
```
Comment (dthaler) [inline specification/gedcom.md:2671]:
```suggestion
can be represented either as a single `FAM` with multiple events (`MARR`, `DIV`, etc.)
```

File: specification/gedcom.md [hunk 3]
```diff
@@ -2663,6 +2663,23 @@ Depending on the maintaining authority, an `EXID` may be a unique identifier for
...
+but that is not always the case.
+
+A couple that separates and then gets together again
+can be represented either as a single `FAM` with multiple events (`MARR`, `DIV`, etc.)
+or as a separate `FAM` for each time together.
+Some user interfaces may display these two in different ways
+and the two admit different semantics in sourcing.
+Neither structure necessarily indicates this kind of temporary separation:
+a single `FAM` with two `MARR` with distinct dates might represent uncertainty about dates
+and a pair of `FAM` with same spouses might be the result of merging multiple files.
```
Comment (tychonievich) [inline specification/gedcom.md]:
```suggestion
A single `FAM` with two `MARR` with distinct dates might also represent uncertainty about dates
and a pair of `FAM` with same spouses might also be the result of merging multiple files.
```

File: specification/gedcom.md [hunk 4]
```diff
@@ -2663,6 +2663,22 @@ Depending on the maintaining authority, an `EXID` may be a unique identifier for
...
+The common case is that each couple has one `FAM` record,
+but that is not always the case.
+
+A couple that separates and then gets together again
+can be represented either as a single `FAM` with multiple events (`MARR`, `DIV`, etc.)
+or as a separate `FAM` for each time together.
+Some user interfaces may display these two in different ways
+and the two admit different semantics in sourcing.
+a single `FAM` with two `MARR` with distinct dates might represent uncertainty about dates
+and a pair of `FAM` with same spouses might be the result of merging multiple files.
```
Comment (tychonievich) [inline specification/gedcom.md:2676]:
```suggestion
A single `FAM` with two `MARR` with distinct dates might also represent uncertainty about dates
and a pair of `FAM` with same spouses might also be the result of merging multiple files.
```

**Pull request merged**